python/README.md

```diff
@@ -0,0 +1,17 @@
+# XenAPI.py usage examples
+
+## How to run the scripts
+
+Each script requires 3 command line arguments:
+
+```
+URL       : an https URL pointing at the server
```

Comment:
Is port number required? If so, should it be noted in this argument description as well as the example below?

Reply:
It's an optional argument. I'll correct the description. In the example it's not necessary because 443 is the default for https.